src/styles/_catalog.scss

```diff
@@ -1,9 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-rows: repeat(auto,200px);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1,200px);
+  }
```

Comment:
you can set` grid-template-rows: repeat(**1**,200px);` by default

Reply:
```suggestion
```